appium/webdriver/webelement.py

```diff
@@ -205,3 +209,18 @@ def set_value(self, value: str) -> T:
         }
         self._execute(Command.SET_IMMEDIATE_VALUE, data)
         return self
+
+    # Override
+    def send_keys(self, *value: str) -> T:
+        """Simulates typing into the element.
+
+        Args:
+            value (str): A string for typing.
+
+        Returns:
+            `appium.webdriver.webelement.WebElement`
+        """
+        keys = keys_to_typing(value)
```

Comment:
keys_to_typing -> to_keys

Reply:
It comes from `selenium`, so couldn't handle it.